angular/projects/admin-nrpti/src/app/mines/mines-detail/mines-detail.component.html

```diff
@@ -1,65 +1,96 @@
 <main class="container-fluid-padding">
-  <div class="d-flex">
-    <h2>Mine Details</h2>
-    <div class="ml-auto">
-      <button class="btn btn-outline-primary mr-1" (click)="navigateBack()">Back</button>
-      <button class="btn btn-primary" (click)="navigateToEditPage()">Edit</button>
+  <h1 id="name" class="heading-bar">{{mine.name}}
+    <span ngbDropdown>
+      <button class="btn btn-primary action-drop-button" id="actionDropdown" ngbDropdownToggle>Actions</button>
+      <div class="dropdown-menu" aria-labelledby="actionDropdown" ngbDropdownMenu>
+        <button class="dropdown-item" [routerLink]="['mines', mine._id, 'upload']">Upload Document</button>
+        <button class="dropdown-item" [routerLink]="['mines', mine._id, 'edit']">Edit</button>
+      </div>
+    </span>
+  </h1>
+  <div class="new-section row d-flex">
+    <h2 class="col-lg-8 col-md-12">Mine Information</h2>
+    <div class="publish-wrapper col-lg-4 col-md-12">
+      <mat-slide-toggle
+        name="publishLng"
+        id="publishLng"
+        (change)="this.togglePublish()"
+        [checked]="isPublished"
+        [disabled]="!canPublish">
+      </mat-slide-toggle>
+      <span class="badge badge-published" *ngIf="isPublished">Published</span>
+      <span class="badge badge-unpublished" *ngIf="!isPublished">Not Published</span>
+      <span class="grey-subtext d-block" *ngIf="!canPublish" >More information is needed to publish this mine.</span>
     </div>
   </div>
+
   <hr>
+
+  <!-- Mine Information -->
+  <section>
+    <div class="row">
+      <!-- column 1/3 -->
+      <div class="col-lg-4 col-md-12">
+        <div class="row label-pair ml-1 mb-2">
+          <div id="permittee"><b>Permittee: </b>{{mine?.permittee || '-' }}</div>
+        </div>
+        <div class="row label-pair ml-1 mb-2">
+          <div id="status"><b>Operating Status: </b>{{mine?.status || '-'}}</div>
+        </div>
+      </div>
+      <!-- column 2/3 -->
+      <div class="col-lg-4 col-md-12">
+        <div class="row label-pair ml-1 mb-2">
+          <div id="permitNumber"><b>Mines Act Permit Number: </b>{{mine?.permitNumbers && mine?.permitNumbers.length > 0 ? mine?.permitNumbers?.join() : '-'}}</div>
```

Comment:
When we have multiple permit numbers, the "join" just stuffs a comma between them. `.join(', ')` will add a space, which is a bit nicer to read.

Second thing: When you have more than a couple, the list can get cluttered. I'm not sure if we're expecting more than a couple normally... if we do it might be nice to restructure the output a bit so it's not just a huge comma separated list. MIght want to ask the UI/UX people about that.

Reply:
Not sure if this helps or not, but on the list page and Core when they have multiple they stack them in a list.

Reply:
Yeah, that's kind of what I was thinking as well. Makes the page a bit longer potentially, but improves the clarity of the list

Reply:
Yeah would be interesting to hear their thoughts on this. For now I'll add that space between the values, thanks!